localgov_guides.module

```diff
@@ -13,12 +13,26 @@ use Drupal\node\NodeInterface;
  */
 function localgov_guides_theme($existing, $type, $theme, $path) {
   return [
-    'guide_contents' => [
+    'guides_contents_block' => [
```

Comment:
Maybe should include namespacing generally for theme templates?

More a question for the future than anything to block this one.

Reply:
I guess the only effect this would have is to the name of templates in the theme should it override them. Is there a good reason why this would be useful?

If we chose to namespace template files then we'll need to change the name of templates in the Services modules as well, so we should create a separate issue for this.

Reply:
I just work on the basic, Drupal convention, that absolutely _anything_ anyone else can also make needs to be prefixed (namespaced) with the module name, because otherwise someone else can also make the same something called guide_contents_block; or guide_field; or whatever. And there's no reason for them not to - before they install your module.

So all paragraphs templates are `paragaphs_` https://git.drupalcode.org/project/paragraphs/-/blob/8.x-1.x/paragraphs.module#L59 https://git.drupalcode.org/project/paragraphs/-/tree/8.x-1.x/templates Just as in core everything in a module will begin with the module name, like `book_` https://git.drupalcode.org/project/drupal/-/blob/9.0.x/core/modules/book/book.module#L57 https://git.drupalcode.org/project/drupal/-/tree/9.0.x/core/modules/book/templates